signer/fulcio/fulcio.go

```diff
@@ -44,104 +53,200 @@ func Signer(ctx context.Context, funcioURL string, oidcIssuer string, oidcClient
 		return nil, err
 	}
 
-	fulcioSigner, err := signature.LoadSigner(key, crypto.SHA256)
-	if err != nil {
-		return nil, err
-	}
+	var raw string
 
-	certResp, err := getCert(fulcioSigner.(*signature.RSAPKCS1v15Signer), fClient, oidcIssuer, oidcClientID)
-	if err != nil {
-		return nil, err
-	}
+	switch {
+	// if we're running in GitHub Actions
```

Comment:
is this an incomplete comment? what's going on if we are running in github actions?